test/unit/report_importer_test.rb

```diff
@@ -10,7 +10,7 @@ class ReportImporterTest < ActiveSupport::TestCase
   end
 
   test 'it should import reports with no metrics' do
-    r = ReportImporter.import(read_json_fixture('report-empty.json'))
+    r = ConfigReportImporter.import(read_json_fixture('report-empty.json'))
```

Comment:
It has been moved, not removed.
https://github.com/theforeman/foreman/pull/2750/files#diff-210611042f12eaeac8d5d9be3c5dc181R8